crates/anvil/core/src/eth/mod.rs

```diff
@@ -321,6 +321,11 @@ pub enum EthRequest {
         with = "sequence"
     )]
     AutoImpersonateAccount(bool),
+
+    /// Registers a signature/address pair for faking `ecrecover` results
+    #[serde(rename = "anvil_recoverSignature", with = "sequence")]
```

Comment:
this could be a bit misleading because this doesnt recover the signature immediately, so perhaps we call this anvil_impersonateSignature instead?